azure-pipelines.yml

```
@@ -41,15 +41,26 @@ steps:
     dockerHubImageName: $(imageName)
     CC_TEST_REPORTER_ID: $(ccReporterID)
 - script: |
+    set -x
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle config --local disable_exec_load true'
     $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake parallel:setup'
-    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,,--format progress --format RspecJunitFormatter --out rspec-results.xml --no-profile]"'
+    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,, -O .azure_parallel]"'
```

I noticed that this was originally `SPEC_OPTS="..."` ... changed in [672c20f](https://github.com/DFE-Digital/manage-courses-backend/commit/672c20fd0ef5ccb1a3df0762b81ab86471eab9e3#diff-fec826feae04e51c0d94076385408bdc). Does it make sense to return it to that more readable form? Unless you're just sick of all this ... :)

---

It appears to be a no-go because `TEST_ENV_NUMBER` needs to be accessed via Rails.env (`ENV`)

i.e. `TEST_ENV_NUMBER` is not defined when the code executed like so ...
```
bundle exec rake parallel:spec SPEC_OPTS="--format RspecJunitFormatter --out rspec-output/rspec-output${TEST_ENV_NUMBER}.xml"
```

Interpolation not poss either

```
 manage-courses-backend 2664-fix-test-output-on-manage-courses-back-end % bundle exec rake "parallel:spec[,, --format RspecJunitFormatter --format documentation --out rspec-output/rspec-output<%= ENV['TEST_ENV_NUMBER'] %>.xml]"
12 processes for 205 specs, ~ 17 specs per process
sh: %=: No such file or directory
sh: %=: No such file or directory
sh: %=: No such file or directory
```

---

Ah. I was just thinking that this is a little bit easy to read:

```suggestion
    $DOCKER_OVERRIDE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec" SPEC_OPTS="-O .azure_parallel"' 
```

No worries, though, this PR has been through enough. :)